README.md

```diff
@@ -133,15 +133,15 @@ we are always happy to help you get started!
 
 ### Opening A Channel
 
-First you need to transfer some funds to `lightningd` so that it can
+You first need to transfer some funds to `lightningd` so that it can
 open a channel:
 
 ```bash
 # Returns an address <address>
 lightning-cli newaddr
 ```
 
-`lightningd` will register the funds once the transaction is confirmed.
+`lightningd` will register the funds, once the transaction is confirmed.
```

Comment:
I don't like the comma here :-) but only a personal thing, I work out of grammar most of the time.

Reply:
Yeah I wouldn't include the comma here either :)